map-chat/javascript/map.js

```diff
@@ -320,9 +320,8 @@ function displayMessageOnMap(msg, olat, olong, sessid, icostr, expir, pokenick)
       }
     });
 
-    if (msg.text) {
+    if (msg.text !== 'undefined') {
```

Comment:
@solderzzc yep, display the location is not bad, but the infoWindow is meaningless, i dont think human can read it. you can try it by yourself. btw, the same info is output in console.debug, too. 
and, if you want a infoWindow, i think there should have a filter.just pop up what you want.